client/component/people.js

```diff
@@ -114,71 +113,57 @@ const AnonymousFaceStyle = StyleSheet.create({
     }
 })
 
-export function FacePile({type='small', userIdList}) {
+export function FacePile({ type = 'small', userIdList }) {
+
     const s = FacePileStyle;
-    const sizeMap = {
-        small: 32,
-        tiny: 24,
-    }
     const size = sizeMap[type] ?? 32;
- 
+
     return <View style={s.outer}>
-        {userIdList.slice(0,3).map((userId, i) => <View key={i} 
-                style={{position: 'relative', marginLeft: i == 0 ? 0 : -(size/4)}}>
+        {userIdList.slice(0, 3).map((userId, i) => <View key={i}
+            style={{ position: 'relative', marginLeft: i == 0 ? 0 : -(size / 4) }}>
             <ProfilePhoto userId={userId} type={type} border />
         </View>)}
     </View>
 }
+
 const FacePileStyle = StyleSheet.create({
     outer: {
         flexDirection: 'row',
     }
 })
 
+export function Byline({ type = 'small', photoType = null, clickable = true, userId, name = null, photo = null, time, subtitleLabel, subtitleParams = {}, underline = false, edited = false }) {
 
-
-
-export function Byline({type='small', photoType=null, clickable=true, userId, name=null, photo=null, time, subtitleLabel, subtitleParams={}, underline=false, edited=false}) {
-    const s = BylineStyle
+    const { tagByline } = useConfig();
```

Comment:
Can we call this bylineTag instead of tagByline?

That follows the convention used for other config slots, where the name starts with the context in which it appears.

Reply:
updated